Realm/ObjectServerTests/RLMObjectServerTests.mm

```diff
@@ -700,18 +639,18 @@ - (void)testAddObjects {
 - (void)testAddObjectsWithNilPartitionValue {
     RLMRealm *realm = [self openRealmForPartitionValue:nil user:self.anonymousUser];
 
+    // Other tests expect the nil partition to be empty so we need to clean up
+    [realm transactionWithBlock:^{
```

Comment:
I feel like this also needs to be called at the end of the test too?

Reply:
Other test with partition value null, are cleaning the database at the beginning of the test as well